interface/interface.py

```diff
@@ -0,0 +1,21 @@
+from PyQt6 import QtWidgets
+from PyQt6.QtWidgets import QApplication, QMainWindow
```

Comment:
for my opinion, these imports are somewhat similar, they cannot be combined?

Reply:
the application will not work